.github/actions/templates/avm-validateModulePSRule/action.yml

```diff
@@ -131,6 +131,35 @@ runs:
         source: "${{ inputs.psrulePath}}/.ps-rule/" # Path to folder containing suppression rules to use for analysis.
         summary: false # Disabling as taken care in customized task
 
+    - name: Run PSRule analysis - Security Pillar Only (Custom Security Pillar)
```

Comment:
Wondering if we could parameterize the different inputs passed from avm.template.module.yml and only have one step here?